project.yaml

```diff
@@ -8,23 +8,62 @@
 version: '3.0'
 
 expectations:
-  population_size: 1000000
+  population_size: 10000
 
 actions:
 
   # Extract data ----
   
   ## Cohort data
-  generate_cohort:
-    run: >
-      cohortextractor:latest generate_cohort
-        --study-definition study_definition
-        --index-date-range "2018-01-01 to 2022-03-01 by month"
-        --output-dir=output
+  generate_study_population_1:
+    run: cohortextractor:latest generate_cohort
+      --study-definition study_definition 
+      --index-date-range "2018-01-01 to 2018-12-01 by month" 
+      --output-dir=output 
+      --output-format=csv
     outputs:
       highly_sensitive:
         cohort: output/input_*.csv
 
+  generate_study_population_2:
+    run: cohortextractor:latest generate_cohort 
+      --study-definition study_definition 
+      --index-date-range "2019-01-01 to 2019-12-01 by month" 
+      --output-dir=output 
+      --output-format=csv
+    outputs:
+      highly_sensitive:
+        cohort: output/input*.csv
+
+  generate_study_population_3:
+    run: cohortextractor:latest generate_cohort 
+      --study-definition study_definition 
+      --index-date-range "2020-01-01 to 2020-12-01 by month" 
+      --output-dir=output 
+      --output-format=csv
+    outputs:
+      highly_sensitive:
+        cohort: output/inpu*.csv
+
+  generate_study_population_4:
+    run: cohortextractor:latest generate_cohort 
+      --study-definition study_definition 
+      --index-date-range "2021-01-01 to 2021-12-01 by month" 
+      --output-dir=output 
+      --output-format=csv
+    outputs:
+      highly_sensitive:
+        cohort: output/inp*.csv
+  
+  generate_study_population_5:
```

Comment:
Just for the initial run please could you remove the unused `generate_study_population_i` as discussed.